spring-boot/autoconfigure/src/main/java/org/togglz/spring/boot/actuate/autoconfigure/TogglzAutoConfiguration.java

```diff
@@ -242,7 +243,7 @@ public void addInterceptors(InterceptorRegistry registry) {
     }
 
     @Configuration
-    @ConditionalOnClass(TogglzDialect.class)
+    @ConditionalOnClass(IProcessorDialect.class)
```

Comment:
Does this remove the warning during application startup?

Reply:
Not really. This is just a bug. Having a @ConditionalOnClass with a class that is part of the same module makes no sense at all. I changed it to test for some class that is part of the thymeleaf library.

The warnings actually is fixed by avoiding that the thymeleaf library is on the class path. So using a Spring Boot starter that does NOT contain this library will remove the warning.

Reply:
I think it was me who implemented it this way :-D interesting. I never thought about this so far. Thanks for the hint. Learned something new :)